openreview/conference/templates/commentProcess.js

```diff
@@ -0,0 +1,90 @@
+function(){
```

Comment:
This deserves a description comment - sends emails to authors, PCs, ACs and reviewers depending on the readers of the note (or of the parent note?).